drivers/test.go

```diff
@@ -530,11 +540,12 @@ func (td *TestDriver) MustCreateAndVerifyMultisigActor(nonce uint64, value abi_s
 		UnlockDuration:        params.UnlockDuration,
 		NumApprovalsThreshold: params.NumApprovalsThreshold,
 
-		PendingTxns: pendingTxMap.Root(),
+		PendingTxns: pendingTxMapRoot,
 	})
 	td.AssertBalance(multisigAddr, value)
 }
 
+// XXX: changes needed here
```

Comment:
@anorth based on the above tests I believe a structure like this is still useful. My guess is that `Treasury` and `RewardsTotal` are not longer required?

Since BlockRewards are now paid directly to miners (instead of being brokered via the reward actor) I think the `Rewards` map could be populated by inspecting each miners vesting store at a specific epoch (I may be way of here)?

Reply:
Treasury is still relevant and easy to get. Drop the reward and total and replace with SimpleSupply, BaselineSupply, LastPerEpochReward from reward actor state.

Change call sites that used this to find the miner's reward to get the miner actor's balance directly, via existing functions.